core/trino-spi/pom.xml

```diff
@@ -188,6 +188,9 @@
                                 <item>
                                     <code>java.annotation.added</code>
                                 </item>
+                                <item>
+                                    <code>java.method.numberOfParametersChanged</code>
```

Comment:
We should probably add a new `beginMerge` method to `ConnectorMetadata` for backwards compatibility... WDYT?

Reply:
Let's consult Trino maintainers on the appropriate method to address this issue